README.md

```diff
@@ -32,8 +32,9 @@ Alpine Linux and is using Docker.
 The installation of this add-on is pretty straightforward and not different in
 comparison to installing any other Hass.io add-on.
 
-**NOTE**: Your Hass.io machine MUST HAVE a static IP for this add-on to function
+**NOTE 1**: Your Hass.io machine MUST HAVE a static IP (not DHCP reserved, but static) for this add-on to function
 properly/fully.
+**NOTE 2**: Since Pi-hole will act as DNS server on your entire local network, you will need to make sure that Hass.io points to a different DNS server (e.g. a public one) to prevent loops (for example, to prevent `NewConnectionError` errors). For more details, see the [Hass.io docs][hass-io-docs-network]. Bottom-line: make sure it doesn't point to itself as DNS server.
```

Comment:
- If this is the case it would only cause other components to be unable to connect till pihole has started. Which means these errors may occur on boot. In that case we better look into a way to delay addon load till pihole has finished starting. (aka fix the problem instead of making notes)
- It isn't a loop though. It shouldn't cause any loops if an addon/plugin doesn't get DNS requests resolved.

Reply:
Your first point sounds great! What if there is another issue? Hassio will be unable to boot, an therefore you’ll be unable to fix the problem as a normal user. Your statement is incorrect in that case. I do agree that part of hassio might be improved, but it isn’t a golden ticket. Hence the problem stays the same.

It isn’t a real loop, although the host becomes a client of a guest... which causes point A.

Reply:
So the main issues is the lack of DNS resolving during boot, which may cause other addons to fail? 
I get your push for forced external DNS now indeed! Which is a shame as it prevents me from filtering dns requests made by addons :'(